survey-runner.tf

```diff
@@ -110,10 +110,20 @@ module "survey-runner-vpc" {
   vpc_cidr_block = "${var.vpc_cidr_block}"
 }
 
+module "ebs_bckup" {
+  source = "github.com/kgorskowski/terraform/modules//tf_ebs_bckup"
+  EC2_INSTANCE_TAG = "RabbitMQ"
+  RETENTION_DAYS   = 10
```

Comment:
we should make this a variable? e.g.. we may want to retain longer in prod than we do in dev?
you could also add a developer default to make this really low to save some £££